apps/api/src/sandbox/services/sandbox.service.ts

```diff
@@ -1278,7 +1278,11 @@ export class SandboxService {
 
     const sshAccess = new SshAccess()
     sshAccess.sandboxId = sandboxId
-    sshAccess.token = nanoid(32)
+    let token: string
+    do {
```

Comment:
Instead of the loop, I suggest:
```typescript
import { customAlphabet as nanoid, urlAlphabet } from 'nanoid'

const token = nanoid(urlAlphabet.replace('_', '').replace('-', ''))(32)
```

(we can make a helper function for this)